neo/IO/Helper.cs

```diff
@@ -9,6 +9,8 @@ namespace Neo.IO
 {
     public static class Helper
     {
+        public const int GroupingSizeInBytes = 16;
```

Comment:
`private`?

Reply:
This also can be needed in code that uses find from a plugin. I have code that needs it, because to search for a prefix one needs to add a 0 byte after every 16 characters to search for a prefix longer than this length.

Reply:
If the find method inserted the 0 bytes after every 16 characters it wouldn’t be needed. If we change that in a subsequent PR we could change this to be internal perhaps, but for now, we need it public.